--- Make-1.00.pius/MANIFEST	Sun Sep 13 16:53:16 1998
+++ Make-1.00/MANIFEST	Sat Dec  4 20:03:33 1999
@@ -5,5 +5,5 @@
 Makefile.PL
 README
 bmake
-pmake
+pmake.pl
 t/make.t
--- Make-1.00.pius/Makefile.PL	Sun Sep 13 16:30:53 1998
+++ Make-1.00/Makefile.PL	Sat Dec  4 19:59:03 1999
@@ -2,7 +2,7 @@
 
 WriteMakefile(
     'NAME'     => 'Make',
-    'EXE_FILES' => ['pmake'],
+    'EXE_FILES' => ['pmake.pl'],
     'clean' => {FILES => "*% *.bak Make/*%"},
     'linkext' => { LINKTYPE => ''},
     'dist'     => { COMPRESS => 'gzip -f', SUFFIX => 'gz' },
--- Make-1.00.pius/README	Sun Apr 11 22:35:33 1999
+++ Make-1.00/README	Sat Dec  4 20:04:02 1999
@@ -9,9 +9,9 @@
 To install 
 
 perl Makefile.PL
-perl -I. pmake 
-perl -Mblib pmake test 
-perl -Mblib pmake install 
+perl -I. pmake.pl 
+perl -Mblib pmake.pl test 
+perl -Mblib pmake.pl install 
 
 No other make required!
 
--- Make-1.00.pius/pmake	Sun Apr 11 20:18:17 1999
+++ Make-1.00/pmake	Thu Jan  1 01:00:00 1970
@@ -1,69 +0,0 @@
-#!/usr/local/bin/perl -w
-use 5.005;  # Need look-behind assertions
-
-use Getopt::Std;
-use Make;
-
-my %opt;
-
-getopts('Dgnpf:j:C:',\%opt);
-
-my $info = Make->new(GNU      => $opt{'g'}, 
-                     Override => { MAKE => "$^X $0" },
-                     Makefile => $opt{'f'}, 
-                     Jobs     => $opt{'j'},
-                     Dir      => $opt{'C'});
-
-if ($opt{'D'})
- {
-  require Data::Dumper;
-  print Data::Dumper::DumperX($info);
-  exit;
- }
-
-if ($opt{'p'})
- {
-  $info->Print(@ARGV);  
-  exit;
- }
-if ($opt{'n'})
- {
-  $info->Script(@ARGV);
- }
-else
- {
-  $info->Make(@ARGV);
- }
-
-=head1 NAME
-
-pmake - a perl 'make' replacement
-
-=head1 SYNOPSIS
-
-	pmake [-n] [-g] [-p] [-C directory] targets
-
-=head1 DESCRIPTION
-
-Performs the same function as make(1) but is written entirely in perl.
-A subset of GNU make extensions is supported.
-For details see L<Make> for the underlying perl module.
-
-=head1 BUGS
-
-=item *
-
-No B<-k> flag
-
-I strongly suspect there are lots more.
-
-=head1 SEE ALSO
-
-L<Make>, make(1)
-
-=head1 AUTHOR
-
-Nick Ing-Simmons 
-
-=cut
-
--- Make-1.00.pius/pmake.pl	Thu Jan  1 01:00:00 1970
+++ Make-1.00/pmake.pl	Sat Dec  4 20:01:32 1999
@@ -0,0 +1,69 @@
+#!/usr/local/bin/perl -w
+use 5.005;  # Need look-behind assertions
+
+use Getopt::Std;
+use Make;
+
+my %opt;
+
+getopts('Dgnpf:j:C:',\%opt);
+
+my $info = Make->new(GNU      => $opt{'g'}, 
+                     Override => { MAKE => "$^X $0" },
+                     Makefile => $opt{'f'}, 
+                     Jobs     => $opt{'j'},
+                     Dir      => $opt{'C'});
+
+if ($opt{'D'})
+ {
+  require Data::Dumper;
+  print Data::Dumper::DumperX($info);
+  exit;
+ }
+
+if ($opt{'p'})
+ {
+  $info->Print(@ARGV);  
+  exit;
+ }
+if ($opt{'n'})
+ {
+  $info->Script(@ARGV);
+ }
+else
+ {
+  $info->Make(@ARGV);
+ }
+
+=head1 NAME
+
+pmake.pl - a perl 'make' replacement
+
+=head1 SYNOPSIS
+
+	pmake.pl [-n] [-g] [-p] [-C directory] targets
+
+=head1 DESCRIPTION
+
+Performs the same function as make(1) but is written entirely in perl.
+A subset of GNU make extensions is supported.
+For details see L<Make> for the underlying perl module.
+
+=head1 BUGS
+
+=item *
+
+No B<-k> flag
+
+I strongly suspect there are lots more.
+
+=head1 SEE ALSO
+
+L<Make>, make(1)
+
+=head1 AUTHOR
+
+Nick Ing-Simmons 
+
+=cut
+
